packages/upswyng-web/src/components/MonthlySchedule.tsx

```diff
@@ -0,0 +1,70 @@
+import Grid from "@material-ui/core/Grid";
+import React from "react";
+import { TScheduleItem } from "@upswyng/upswyng-core";
+import Typography from "@material-ui/core/Typography";
+
+const groupByRRuleText = (items: TScheduleItem[]) =>
+  items.reduce(
+    (groupedItems: { [key: string]: TScheduleItem[] }, item: TScheduleItem) => {
```

Comment:
```suggestion
    (groupedItems: Record<string, TScheduleItem[]>, item: TScheduleItem) => {
```

These are equivalent but the linter has been yelling at me for the object version... weird.

Reply:
can do. [upderped](https://github.com/CodeForBoulder/upswyng/pull/262/commits/00a462902fbdf00bb1d9b461dd890db2f9df198d)